docker/px4-dev/Dockerfile_base

```diff
@@ -23,16 +24,23 @@ RUN apt-get update && apt-get -y --quiet --no-install-recommends install \
 		gnupg \
 		gosu \
 		lcov \
+		libfreetype6-dev \
+		libpng-dev \
+		lsb-release \
 		make \
 		ninja-build \
 		openjdk-8-jdk \
 		openjdk-8-jre \
 		openssh-client \
+		pkg-config \
 		python-pip \
 		python-pygments \
 		python-setuptools \
+		python3-dev \
```

Comment:
Can we pick a single version of python?

Reply:
Actually I was considering moving this to the ROS container only. I am still on trial and error trying to find the problem with pip.

Reply:
We depend on python3 to ROS2.

Reply:
So could we finally go python3 everywhere?

Reply:
I don't think python3 works out of the box with ROS kinetic. At least it didn't used to.

Reply:
@dagar according to http://www.ros.org/reps/rep-0003.html, Melodic still needs Python 2.7, as it is a targeted landguage. So no, we cannot drop Python 2 for now.